packages/yew-router/src/lib.rs

```diff
@@ -55,17 +55,18 @@
 
 extern crate self as yew_router;
 
-#[doc(hidden)]
-#[path = "macro_helpers.rs"]
-pub mod __macro;
 pub mod components;
+pub mod history;
 mod routable;
 pub mod router;
 mod service;
 pub(crate) mod utils;
 
 pub use service::*;
 
+#[doc(hidden)]
+pub mod hidden;
```

Comment:
Is there a reason why you named it to `hidden` from `macro_helpers`?

Reply:
Not particularly - I didn't specifically look to rename it, but initially `macro_helpers` got removed because I wasn't using any of it, and then what is now "hidden.rs" came later. I named it "hidden" because that's the convention I've seen elsewhere.

Reply:
Yew uses `__macro` name/prefix for items used only in macros so we should start consistent with that. With that module name, `macro_helpers.rs` sounds like a better file name.

Reply:
Sure. I think `hidden` is used because it's short so it's not too ugly when you have to use absolute paths to refer to everything from a macro.